clang/lib/AST/DeclPrinter.cpp

```diff
@@ -1189,6 +1189,16 @@ void DeclPrinter::printTemplateParameters(const TemplateParameterList *Params,
   Out << '>';
   if (!OmitTemplateKW)
     Out << ' ';
+
+  if (const Expr *RequiresClause = Params->getRequiresClause()) {
```

Comment:
If I read the code correctly, looks like we can move this code to Line 1190 (just immediately before the  above `if (!OmitTemplateKW)`)? Then we can get rid of all the `OmitTemplateKw` logic inside this if branch.

Reply:
Yeah, you're right, will do that shortly.